deprecated/deprecated-functions.php

```diff
@@ -63,7 +64,11 @@ function dokan_depricated_widget_classes() {
 add_action( 'woocommerce_before_main_content', 'dokan_depricated_widget_classes' );
```

Comment:
What is the replacement of the following action
```php
add_action( 'woocommerce_before_main_content', 'dokan_depricated_widget_classes' );
```
Probable replacement could be https://github.com/getdokan/dokan/blob/2a7c914bff8ffb9f840a7bd03aa0551f6a63addd/includes/Widgets/Manager.php#L19